### PR TITLE
:sparkles: New Layout

### PR DESCRIPTION
**Pull request by KipandaJr**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206628060265901
  - https://app.asana.com/0/0/1206628060736062

### CODE REVIEW

File: components/account-info/movement-network.tsx [hunk 1]
```diff
@@ -49,11 +56,10 @@ const MovementNetwork: FC = () => {
...
-        alignItems="center"
-        border="0.25rem solid"
         onClick={handleOpenMenu}
-        borderColor={isOpen ? '#0053DB33' : 'transparent'}
+        borderRadius="xs"
+        alignItems="center"
+        bg={isOpen ? `${colors.primary}14` : 'container'}
       >
         <Box
           color="white"
```
Comment (MarioBatalha) [inline components/account-info/movement-network.tsx:65]: Use theme value

File: components/account-info/movement-network.tsx [hunk 2]
```diff
@@ -67,29 +73,36 @@ const MovementNetwork: FC = () => {
...
       {isOpen && (
         <Motion
           right="0"
-          top="5rem"
+          top="3rem"
           zIndex={4}
           width="14.5rem"
           initial="closed"
-          border="1px solid"
           borderRadius="1rem"
```
Comment (MarioBatalha) [inline components/account-info/movement-network.tsx:101]: Use theme value, in this case "m"

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (git-marcopitra) [inline yarn.lock:1]: remove this file

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Review by git-marcopitra — APPROVED** [review]

File: components/layout/sidebar/menu-item/accordion-item.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <Box
+      mx="auto"
+      width="100%"
+      display="flex"
+      borderRadius="m"
+      onClick={goToPath}
+      opacity={disabled ? 0.3 : 1}
+      transition="all 350ms ease-in-out"
+      nHover={disabled ? {} : { color: 'primary' }}
```
Comment (atilasousa) [inline components/layout/sidebar/menu-item/accordion-item.tsx:26]: maybe this way would be cleaner

nHover={disabled && { color: 'primary' }}

Comment (git-marcopitra) [reply components/layout/sidebar/menu-item/accordion-item.tsx:26]: `{...(disabled && { nHover: { color: 'primary' })}`

**Review by MarioBatalha — APPROVED** [review]

**Review by atilasousa — APPROVED** [review]

**Pull request merged**